### ISSUE TITLE
No colours or icons on running from git bash on windows

### ISSUE THREAD

**Issue by olivif** [IonicaBizau/namly#3]
This is what it looks like when I run from git bash on windows. 

![image](https://cloud.githubusercontent.com/assets/7736961/11913895/67c23ae0-a673-11e5-9a11-9757aaa63bef.png)

I ran the same test in index.js where I can see some names are not available so I was expecting similar visuals to https://github.com/IonicaBizau/namly/blob/master/README.md

**Comment by IonicaBizau** [comment]
Hmm, looks like Git Bash doesn't support Emoji, nor ANSI colors. That's probably expected since Git Bash uses the built in terminal which is rolled into the cmd prompt.

Could you try using [_Cygwin Terminal_](https://www.cygwin.com/)?

**Comment by olivif** [comment]
Yeah, I think you might be right. Running from windows cmd directly gave me that same thing :( 

Trying Cygwin now.

Could it just use basic colours in git bash? Red/green type of thing. I'm thinking that most windows users won't really switch to Cygwin only to run a couple of packages :)

**Comment by olivif** [comment]
Same thing from Cygwin on Windows 

![image](https://cloud.githubusercontent.com/assets/7736961/11914040/a1dc6960-a675-11e5-97f9-518f574e08b5.png)

**Comment by IonicaBizau** [comment]
@olivif Oh, I see! Any ideas how ANSI colors are interpreted on Windows (you should know that!)?

[`couleurs`](https://github.com/IonicaBizau/node-couleurs) is waiting for you to improve the Windows support. :grinning:

**Comment by olivif** [comment]
@IonicaBizau look at this beauty running on Windows (couleurs example)! Haha!

![image](https://cloud.githubusercontent.com/assets/7736961/11914079/5c1a634a-a676-11e5-916c-495d0493ca34.png)

You're right. Let me see what I can dig up, fork couleurs and see if we can make that better :)

**Comment by IonicaBizau** [comment]
@olivif Ohhh! :scream: Definitely!

**Comment by olivif** [comment]
@IonicaBizau btw I will file an issue over there so we can track it. And then we can come back to this one :smile:

**Comment by IonicaBizau** [comment]
@olivif Perfect! :+1:

**Comment by olivif** [comment]
@IonicaBizau will bumping the node-couleurs version of this help now?

**Comment by IonicaBizau** [comment]
@olivif Exactly. It should fix the colors. :smile:

**Comment by olivif** [comment]
Hmm, I don't see a direct dependency. Is it in one of the other packages?

**Comment by IonicaBizau** [comment]
It's in the `emoji-logger`.
